src/compute/src/compute_state.rs

```diff
@@ -38,6 +38,15 @@ use crate::logging;
 use crate::logging::materialized::ComputeEvent;
 use crate::sink::SinkBaseMetrics;
 
+/// Extra context shared within a single `Worker`
+/// instance.
+/// Should be kept cheaply-cloneable.
+#[derive(Clone)]
+pub struct SharedComputeState {
+    /// The debug level to set librdkafka to
+    pub librdkafka_debug: tracing::Level,
+}
```

Comment:
nit: "shared" makes me think `Arc`, but this is just directly cloned.

Reply:
"cheaply-cloneable" to me means it contains `Copy` types or `Arc`-wrapped types

Reply:
is `CommonComputeState` reasonable?